lib/Extension/Debug/Model/DocumentorInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Phpactor\Extension\Debug\Model;
+
+interface DocumentorInterface
```

Comment:
we don't use suffixes, this can be just `Documentor`

Reply:
Fixed